### ISSUE TITLE
pass function in lightGallery options

### ISSUE THREAD

**Issue by YellowLabIT** [sachinchoolur/lightGallery#731]
Hello to everybody
is there a way to pass something like a function in lightGallery options?

` $lg.lightGallery({
                cssEasing: 'cubic-bezier(0.25, 0, 0.25, 1)',
                controls: false,
                closable: false,         
                enableSwipe: swipeOrNotToSwipe(),
                otherOptions: 
});

function swipeOrNotToSwipe() {
if (zoom > 1) enableSwipe: false
else (enableSwipe true)
} `
so this is not possible because var _zoom_ is created inside onAfterOpen.lg
is there another way to modify lightGallery options after lightGallery was open?

Basically I want to disable swipe and drag when image is zoomed (we are not using lg-zoom module) so this assure you can swipe and drag actual image without changing slide.

Another way using lg-zoom module (maybe more simple?), is to add .lg-zoomed class while my var _zoom_ is > 1. i've just tested it and when you pinchzoom and swipe/drag you swipe/drag the image and not through the gallery. Awesome BUT lg-zoom module conflict whit my library. 
Is there a way to use class .lg-zoomed also if i set zoom: false in lightGallery options?

thankyou :)
thankyou :)
thankyou :)

Tiziano

**Comment by YellowLabIT** [comment]
Finally we find an alternative solution, 
by just adding a very simple line of code.
the solution worked for us is:
when image is zoomed (directly pinch zoom on .lg), add class .no-touch to .lg-inner else class .no-touch is removed by jquery.
in css
`.no-touch { pointer-events: none }`
this stop touch event to lg-inner and all sub-elements insisde.

**Comment by piemme74** [comment]
Please can you paste the code in which you add .no-touch ?
thanks a lot
